sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentServiceResponse.java

```diff
@@ -83,7 +84,7 @@ public RxDocumentServiceResponse(DiagnosticsClientContext diagnosticsClientConte
             return InternalConstants.ResourceKeys.CLIENT_ENCRYPTION_KEYS;
         }
 
-        throw new IllegalArgumentException("c");
+        return InternalConstants.ResourceKeys.DOCUMENTS;
```

Comment:
I wonder instead of using classType, would ResourceType be a better here

Reply:
Possible - but don't want to change more than necessary - PR is invasive enough already.